.github/workflows/cla.yml

```diff
@@ -19,5 +19,5 @@ jobs:
         with:
           path-to-signatures: 'signatures/cla.json'
           path-to-cla-document: 'https://github.com/exadel-inc/face-recognition-service/blob/develop/CLA.md'
```

Comment:
Please update path to CLA

Reply:
We will update CLA.md through develop branch, won't we? So there always will be the actual version.